src/login/user-runtime-dir.c

```diff
@@ -277,7 +277,7 @@ static int apply_tmpfs_quota(
                 req.dqb_valid = QIF_BLIMITS;
                 req.dqb_bsoftlimit = req.dqb_bhardlimit = v;
 
-                r = RET_NERRNO(quotactl_fd(fd, QCMD_FIXED(Q_SETQUOTA, USRQUOTA), uid, &req));
+                r = RET_NERRNO(quotactl_fd_with_fallback(fd, QCMD_FIXED(Q_SETQUOTA, USRQUOTA), uid, &req));
```

Comment:
Nah, support for tmpfs quota is added in kernel 6.4, while the fallback provides compat for kernel < 5.14. no need for extra churn here.

Reply:
i mean: if this is changes as @yuwata it's easy to just grep for quotactl_fd and see that everything uses the wrapper, instead of having to look at each case to identify the cases where it's actually fine to use the non-wrapper version...

Also, as tiny benefit it means we can get rid of the RET_NERRNO()...

Reply:
(also, this wrapper doesn't have this nutty 'caddr_t' argument)

Reply:
I'm still not convinced. This really confuses me when the fallback path is essentially useless...

Reply:
OK. The change is dropped now. I do not have any strong opinion to push the change.